Drop-In/src/main/java/com/braintreepayments/api/CardDetailsFragment.java

```diff
@@ -45,6 +46,7 @@ public void onCreate(Bundle savedInstanceState) {
             dropInRequest = args.getParcelable("EXTRA_DROP_IN_REQUEST");
             configuration = args.getParcelable("EXTRA_CARD_FORM_CONFIGURATION");
             cardNumber = args.getString("EXTRA_CARD_NUMBER");
+            isTokenizationKeyAuth = args.getBoolean("EXTRA_AUTH_IS_TOKENIZATION_KEY");
```

Comment:
CardDetailsFragment needs to know if the authorization is a Tokenization key to determine if vaulting is possible and if it should display the save card check box. It was previously accessing this information from the authorization fields on `DropInRequest`. To avoid merchants having to set authorization on both the `DropInRequest` and in the `DropInClient` constructor, I'm passing the auth type as an extra to the fragment, so the auth from `DropInClient` is the single source.

Reply:
Doesn't the `DropInRequest` already have access to the auth? Would it make more sense to instead expose it on the `DropInRequest` than to use an extra?

Reply:
`DropInRequest` had access to this auth when it was set by the merchant through the `clientToken` or `tokenizationKey` setters. `DropInClient` also needs authorization to be instantiated, so without this change the merchant has to set the same parameter in both the request and the client. 

I opted to remove the duplicate step for merchants and pass auth as an arg (also to avoid the possibility that different authorizations are set in the request and the client). 

Alternatively - we could keep the authorization setters on `DropInRequest` and have `DropInClient` access the auth from the `DropInRequest` on instantiation. I went with the auth in the client constructor to match `BraintreeClient`, but it could go either way here.

Reply:
`CardDetailsFragment` doesn't have access to `DropInClient` so that seems trickier. Is there another way to do this? I could put the boolean in the view model, but that doesn't really seem like the right place for it either.

Reply:
Oh nah you're right 🤦‍♂️ that makes total sense.

Reply:
> DropInRequest had access to this auth when it was set by the merchant through the clientToken or tokenizationKey setters. DropInClient also needs authorization to be instantiated, so without this change the merchant has to set the same parameter in both the request and the client.

I'm a little confused. How come a merchant can either set the auth credentials on both the DropInClient & the DropInRequest?

Reply:
Ah yes - to clarify, those authorization methods on `DropInRequest` existed in the previous version of Drop-in. This PR removes those methods on `DropInRequest` in favor of instantiating `DropInClient` with authorization.

Reply:
Got it. Removing those on DropInRequest makes more sense to me now. Thank you for the follow up 👍